web/application/controllers/Apis.php

```diff
@@ -1035,52 +1036,124 @@ public function airqoPlacesJobUpdate()
     }
 
 
-public function placeForecast()
+// public function placeForecast()
+//     {   
+//         $response = array();
+//         $this->ApisModel->init();
+//         $this->form_validation->set_rules('lat', 'Place latitude', 'trim|required');
+//         $this->form_validation->set_rules('lng', 'Place longitude', 'trim|required');
+//         $this->form_validation->set_rules('api', 'API', 'trim|required');
+//         if ($this->form_validation->run() == false) {
+//             echo $this->ApisModel->api_error();
```

Comment:
we need to minimised commented out code, is this for future use or you are disabling a feature? If not, please just remove. @adongot @ja-odur